include/parcels.h

```diff
@@ -66,15 +81,35 @@ static inline ErrorCode temporal_interpolation_linear(float x, float y, int xi,
   err = search_linear_double(time, f->tdim, f->time, &(f->tidx));
   if (f->tidx < f->tdim-1 && time > f->time[f->tidx]) {
     t0 = f->time[f->tidx]; t1 = f->time[f->tidx+1];
-    err = spatial_interpolation_bilinear(x, y, i, j, f->xdim, f->lon, f->lat,
-                                        (float**)(data[f->tidx]), &f0);
-    err = spatial_interpolation_bilinear(x, y, i, j, f->xdim, f->lon, f->lat,
-                                        (float**)(data[f->tidx+1]), &f1);
+    if (interp_method==0){
```

Comment:
Comparing to integer values here is confusing and makes it hard to guess what value encapsulates which method. How about adding an `enum` similar to `ErrorCode` so that you can write `interp_method==NEAREST` here?